app/test_main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+def test_check_valid_url(
+        mocked_valid_google_url: mock.MagicMock
+) -> None:
```

Comment:
The test name `test_check_valid_url` could be more descriptive. Consider renaming it to something like `test_valid_url_calls_valid_google_url` to clearly indicate what aspect of the function is being tested.